package.json

```diff
@@ -36,6 +36,7 @@
     "react-native-reanimated": "~2.3.1",
     "react-native-safe-area-context": "3.3.2",
     "react-native-screens": "~3.10.1",
+    "react-native-svg": "^12.3.0",
```

Comment:
there's this duplicate `react-native-svg` at the bottom but I can't put this comment on it 😅

Reply:
Removed the older one; thanks!

Reply:
I think we may have wanted the older version.  Spinning this up, I get the following warning:

> Some dependencies are incompatible with the installed expo package version:
> - react-native-svg - expected version: 12.1.1 - actual version installed: 12.4.4

Alternatively, we could get the expo 45 changes merged in here (I'm assuming that's maybe where the newer version comes from).  I just tested and approved that PR, but it hasn't been merged in yet.

Reply:
Let's get the 45 changes in and see what happens :+1: